Autopilot/AttitudeManager/Src/SendInstructionsToSafety.cpp

```diff
@@ -1,5 +1,6 @@
 #include "SendInstructionsToSafety.hpp"
 #include "Interchip_A.h"
+#include <stdint.h>
```

Comment:
use <cstdint> rather than the deprecated c equivalent.

Reply:
would that be <cstdint>?